genai-perf/genai_perf/parser.py

```diff
@@ -436,7 +436,7 @@ def _set_artifact_paths(args: argparse.Namespace) -> argparse.Namespace:
         if args.service_kind == "openai":
             name += [f"{args.service_kind}-{args.endpoint_type}"]
         elif args.service_kind == "triton":
-            name += [f"{args.service_kind}-{args.backend.to_lowercase()}"]
+            name += [f"{args.service_kind}-{args.backend}"]
```

Comment:
I don't know why this wasn't causing an issue previously, since this code has been here for months.

Args.backend is a string, so there is no to_lowercase() function. This line returned an error when using the default backend value. Moreover, the enum helper function means all of the strings are already lowercase.